org.adempiere.base/src/org/compiere/wf/MWFProcess.java

```diff
@@ -333,6 +359,15 @@ else if (activityState.isAborted() && !WFSTATE_Terminated.equals(closedState))
 	 * 	@param trxName transaction
 	 */
 	public void checkCloseActivities(String trxName) {
+		checkCloseActivities(isDisallowAutoStartNextNode(), trxName);
+	}
+
+	/**************************************************************************
+	 * 	Update process status based on status of activities.
+	 * 	@param isDisallowStartNextNode - CLDE
+	 * 	@param trxName transaction
+	 */
+	public void checkCloseActivities(boolean isDisallowStartNextNode, String trxName) {

```

Comment:
what is the use case for this isDisallowStartNextNode change  ? Looks hackish

Reply:
Yes I agree, with that flag you can control the workflow process, to allow to execute only one node at a time, do not start the next node, rather wait for the next event that will continue the workflow. 

For now it is just in java, maybe it is redundant since that should be controlled by adding Condition to the Transition. Should I remove it, or would it make sense in e.g. the workflow configuration, wdyt?